frontend/app/features/kanban/hooks/useTaskFilters.ts

```diff
@@ -0,0 +1,58 @@
+import { useGetAllTasksQuery } from "@store/api/taskSlice";
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+
+export interface TaskFilters {
+  status: string;
+  createdBy: string;
+  assignedTo: string;
+}
+
+export const useTaskFilters = () => {
+  const { register, watch, setValue } = useForm<TaskFilters>({
+    defaultValues: {
+      status: "ALL",
+      createdBy: "ALL",
+      assignedTo: "ALL",
+    },
+  });
+
+  const [activeFilters, setActiveFilters] = useState<TaskFilters>({
+    status: "ALL",
+    createdBy: "ALL",
+    assignedTo: "ALL",
+  });
+
```

Comment:
why there are 2 states for filter????, use form is enough and already should have active filters so please remove the other